packages/data-transport-layer/src/services/run.ts

```diff
@@ -49,6 +58,7 @@ interface Bcfg {
       ),
       defaultBackend: config.str('default-backend', 'l1'),
       sentryDsn: config.str('sentry-dsn'),
+      sentryTraceRate: config.uint('sentry-trace-rate', 0.05),
```

Comment:
You'll want to parse it as `config.ufloat('sentry-trace-rate', 0.05)` instead of `uint`
https://github.com/bcoin-org/bcfg/blob/1216c775832c2e138ae6451912a27ba496dc4386/lib/config.js#L433

Reply:
good catch, fixed!

Reply:
good catch, fixed!